tool/lib/utils.dart

```diff
@@ -84,6 +84,17 @@ class CliCommand {
     );
   }
 
+  factory CliCommand.git(
+    String args, {
+    bool throwOnException = true,
+  }) {
+    return CliCommand._(
+      exe: 'git',
+      args: args.split(' '),
```

Comment:
this might fail if anyone tries to use something in quotes. i.e. ` git commit -m "a message"`

Reply:
fixed the one case of that.